src/js/player.js

```diff
@@ -256,11 +256,11 @@ class Player extends Component {
     if (options.plugins) {
       const plugins = options.plugins;
 
-      Object.getOwnPropertyNames(plugins).forEach(function(name) {
+      Object.keys(plugins).forEach(function(name) {
         if (typeof this[name] === 'function') {
           this[name](plugins[name]);
         } else {
-          log.error('Unable to find plugin:', name);
+          throw new Error(`plugin "${name}" does not exist`);
```

Comment:
I feel like mis-configuration of the player is something we ought to be really loud about.